### PR TITLE
Added models and usage documentation

### PR DESCRIPTION
**Pull request by charlwillia6**
image
image
image
image

### CODE REVIEW

File: docs/tab-autocomplete.md [hunk 1]
```diff
@@ -31,8 +31,22 @@ PearAI supports tab autocomplete, which predicts and suggests what you would typ
...
 
+- You can use [Supermaven](https://supermaven.com/) for tab autocomplete. It is currently one of the best autocomplete AI on the market, and provides a free tier. You can get started by installing Supermaven directly as an extension within PearAI.
+  
   ![Supermaven extension](../static/img/supermaven.png)
+
+- Also, you can use [Ollama](https://ollama.ai/) local models.  Download the model that you want and add it to your config.json file. Examples:
+
+  ```json
+  "tabAutocompleteModel": {
+      "title": "Codegemma",
```
Comment (Fryingpannn) [inline docs/tab-autocomplete.md:44]: how are these local models in your experience if you have tried them? nang had tried ollama and said it was ass XD

Comment (charlwillia6) [reply docs/tab-autocomplete.md:44]: Honestly, Ollama local models depend on how much vram your computer has. If you aren't running a computer with a good GPU, they are gonna be slow and I think it does affect the quality of the code completions. 

I have had good quality code completions with StarCoder2 and Deep-seek. However, I couldn't run Deep-seek 2, which I heard is even better than the original. 

I do know from some discussion boards that many devs are using local models for tab auto completions, so I wanted to add this to the documentation as another option, and it lets users know that it is an option, and that PearAI does indeed support local models. 

Oh, but I had crappy luck with Codegemma. It sucked. So, I will update my PR and remove it. I am not sure why I through that one in there.

Comment (charlwillia6) [reply docs/tab-autocomplete.md]: @Fryingpannn PR has been updated.  I left StarCoder2 as an example Ollama model that a user could configure.  Really, Codestral has been what I have been using, and it is great!  I recommend it over our other choices.  But it is hard to find a good model that is tab autocomplete specific.  I have found it is hard to find reliable reviews.

**Comment by charlwillia6** [comment]
@Fryingpannn - I removed the edits to the tab autocomplete section so that the only changes added this PR are the model usage additions.  That way we can get this PR approved.  I will open a separate PR for the tab autocomplete changes.

**Comment by jpan8866** [comment]
there's been some updates recently regarding the usage. We are measuring by tokens generated (prompt + llm response), calculated directly using the provider's tokenizer. The hard limit is $15/month, after which a top-up is required. We've added a top-up feature (see https://github.com/trypear/pear-landing-page/pull/307) that allows a user to add credits to their account. These credits do not expire and are used only when a user hits their monthly limit. This feature is already in prod

Also might be good to clarify that only requests that use the PearAI server counts as "usage".

**Comment by charlwillia6** [comment]
I can add this @jpan8866 https://github.com/trypear/pearai-documentation/pull/18#issuecomment-2420731359

**Comment by charlwillia6** [comment]
> I can add this @jpan8866 [#18 (comment)](https://github.com/trypear/pearai-documentation/pull/18#issuecomment-2420731359)

@jpan8866 @Fryingpannn Updated!

**Comment by charlwillia6** [comment]
I forgot to add something in regards to: only requests that use the PearAI server counts as "usage". I will update the PR today.

File: docs/models-and-usage.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+We understand that usage limits can sometimes be a bit confusing. We're always striving to improve PearAI and provide the best possible AI experience. If you have any questions or concerns, please don't hesitate to contact us by [email](mailto:pear@trypear.ai) or join our [Discord](https://discord.gg/7QMraJUsQt).
+
+## Available models
+### PearAI server
+- Claude 3.5 Sonnet latest
+- Claude 3 Haiku
+- GPT4o latest
+- GPT4o Mini (currently annual subscribers only)
+- GPT4o Preview (currently annual subscribers only) (high cost)
```
Comment (charlwillia6) [inline docs/models-and-usage.md:35]: These should be o1.

**Review by Fryingpannn — APPROVED** [review]

File: docs/models-and-usage.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+title: 🚀 Costs & Available Models
+description: PearAI Tips Usage Costs and Available Models
+keywords: [pearai, models, usage, tips, recommendations]
+---
+
+# 🚀 Usage Costs & Available Models
+
+## What are the usage costs? (PearAI Server only)
+
```
Comment (charlwillia6) [inline docs/models-and-usage.md:10]: Ouput should be output

**Pull request merged**